Buildings/Fluid/HeatExchangers/Validation/WetCoilEffectivenessNTU.mo

```diff
@@ -0,0 +1,298 @@
+within Buildings.Fluid.HeatExchangers.Validation;
+model WetCoilEffectivenessNTU
+  "Model that validates the wet coil effectiveness-NTU model"
+  extends Modelica.Icons.Example;
+
+  package Medium_W = Buildings.Media.Water;
+  package Medium_A = Buildings.Media.Air;
+
+  constant Modelica.SIunits.AbsolutePressure pAtm = 101325
+    "Atmospheric pressure";
+
+  parameter Modelica.SIunits.Temperature T_a1_nominal=
+    Modelica.SIunits.Conversions.from_degF(42)
+    "Inlet water temperature";
+  parameter Modelica.SIunits.Temperature T_a2_nominal=
+    Modelica.SIunits.Conversions.from_degF(80)
+    "Inlet air temperature";
+  parameter Modelica.SIunits.Temperature T_b1_nominal_wet=
+    273.15+11.0678
+    "Outlet water temperature in fully wet conditions";
+  parameter Modelica.SIunits.Temperature T_b2_nominal_wet=
+    273.15+13.5805
+    "Outlet air temperature in fully wet conditions";
+  parameter Real X_w_a2_nominal_wet = 0.0173
+    "Inlet water mass fraction in fully wet conditions";
+  parameter Modelica.SIunits.ThermalConductance UA_nominal = 4748
+    "Total thermal conductance at nominal flow, from textbook";
+  parameter Modelica.SIunits.MassFlowRate m1_flow_nominal = 3.78
+    "Nominal mass flow rate of water";
+  parameter Modelica.SIunits.MassFlowRate m2_flow_nominal = 2.646
+    "Nominal mass flow rate of air";
+  parameter Types.HeatExchangerConfiguration hexCon=
+    Types.HeatExchangerConfiguration.CrossFlowStream1MixedStream2Unmixed
+    "Heat exchanger configuration";
+  Buildings.Fluid.Sources.Boundary_pT sinAir(
+    redeclare package Medium = Medium_A,
+    use_p_in=false,
+    nPorts=3)
+    "Air sink"
+    annotation (Placement(transformation(extent={{-180,-72},{-160,-52}})));
+  Sources.MassFlowSource_T souAir(
+    redeclare package Medium = Medium_A,
+    m_flow=m2_flow_nominal,
+    T=T_a2_nominal,
+    use_Xi_in=true,
+    nPorts=1)
+    "Air source"
+    annotation (Placement(transformation(extent={{140,-70},{120,-50}})));
+  Buildings.Fluid.Sources.Boundary_pT sinWat(
+    redeclare package Medium = Medium_W,
+    nPorts=3)
+    "Sink for water"
+    annotation (Placement(transformation(extent={{50,10},{30,30}})));
+  Modelica.Blocks.Sources.CombiTimeTable X_w2(
+    table=[0,0.0035383; 1,0.01765],
+    timeScale=100) "Water mass fraction of entering air"
```

Comment:
Just to double check with you that the textbook data is in kg/kg_total and not kg/kg_dryAir.